_data/speakers.yml

```diff
@@ -223,4 +224,42 @@ dns2utf8:
       link: http://estada.ch
   desc:
 
+llogiq:
+  name: Andre 'llogiq' Bogus
+  thumbnailUrl: llogiq.jpg
+  links:
+    - icon: website
+      link: https://llogiq.github.io/
+    - icon: github
+      link: https://github.com/llogiq
+    - icon: twitter
+      link: https://twitter.com/llogiq
+  desc: |
+    The quintessential Rust bard Musician by heart, programmer by trade, Rustacean, Lint maven, nags weekly on rust-users and reddit.
+
+imperio:
+  name: Guillaume Gomez
+  thumbnailUrl: imperio.jpg
+  links:
+    - icon: github
+      link: https://github.com/GuillaumeGomez
+    - icon: website
+      link: https://blog.guillaume-gomez.fr
+  desc: |
+    Young IT engineer, enthusiast open source contributor, Rust reviewer, gtk-rs organization owner and GNOME foundation member.
+
+slomo:
```

Comment:
I know :slightly_smiling_face: I stuck to the nickname convention since we already used it for the existing workshop hosts (and since three of them are also part of the collection of organizers and helpers where we use nicknames as identifiers)